### DISCUSSION TITLE
Who holds the Guinness 🏆 for smashing 🎸 on stage?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1128] [Galaxy Trivia] [answered]
**Bonus:** How many guitars have been smashed?

**Comment by chinchern** [comment]
Dave Browne.

**Comment by peterbaccaro** [comment]
**Matt Bellamy** of Muse holds the Guinness World Record for the most guitars smashed on stage.  He smashed **140 guitars** on Muse's 2004 tour.

**Comment by thuongtruong1009** [comment, marked as answer]
Jimi Hendrix held the Guinness World Record for the most guitars smashed on stage during a single tour. During his 1967 tour, Hendrix reportedly smashed 52 guitars. Please note that records may have changed since then, and you may want to check the latest Guinness World Records for the most up-to-date information.
